src/eventEmitter.ts

```diff
@@ -6,7 +6,9 @@ export class EventEmitter {
   emit(type: string, data: any) {
     const handlerFuncs = this.handlers[type];
     if (!handlerFuncs || handlerFuncs.length === 0) {
-      return;
+      throw new Error(
```

Comment:
Not having any handlers registered is a valid case, we don't want to start throwing errors if the builder doesn't care about listening to the event.

Reply:
Right, I see what you mean